plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -656,6 +657,18 @@ protected String tableDefinitionForQueryLoggingCount()
                 ")";
     }
 
+    @Override
+    protected OptionalInt maxTableNameLength()
+    {
+        return OptionalInt.of(256);
```

Comment:
I don't see explicit checks in KuduMetadata yet. Is the goal of this change only to enforce limit on our side for connectors that we know will silently truncate table names? Or do we want to always enforce the limits?

Reply:
No change requested - just a question to understand scope of change.

Reply:
The current goal is just ensuring the failure and the place (remote or connector) is out of scope.